database/migrations/2023_03_20_223400_create_votes_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('votes', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('nominee_id')->references('id')->on('nominees');
```

Comment:
Also mám tady použít cascade? Když chci smazat nominee a má votes, tak mi to logicky vyhodí error